guides/using_hugging_face_integrations.md

```diff
@@ -0,0 +1,158 @@
+
+Let's combine the notion of running things in parallel with the Spaces integration. The [GPT-J-6B](https://huggingface.co/spaces/mrm8488/GPT-J-6B) Space demos a model that generates text using a model called GPT-J. The [T0pp](https://huggingface.co/spaces/akhaliq/T0pp) Space demos another generative model called T0pp. Let's see how to combine both into one.
+
+```python
+import gradio as gr
+
+iface1 = gr.Interface.load("spaces/mrm8488/GPT-J-6B")
+iface2 = gr.Interface.load("spaces/akhaliq/T0pp")
+
+iface3 = gr.mix.Parallel(
+  iface1, iface2, 
+  examples = [
+    ['Which country will win the 2002 World Cup?'],
+    ["A is the son's of B's uncle. What is the family relationship between A and B?"],
+    ["In 2030, "],
+  ])
+  
+iface3.launch()
+```
+
+`iface1` and `iface2` are loading existing Spaces. Then, with `Parallel`, you can run the interfaces parallelly. When you click submit, you will get the output for both interfaces. This is how the demo looks like:
+
+<iframe src="https://hf.space/gradioiframe/osanseviero/mix_match_gradio/+" frameBorder="0" height="450" title="Gradio app" class="container p-0 flex-grow space-iframe" allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"></iframe>
+
+Although both models are generative, you can see that the way both models behave is very different. That's a powerful application of `Parallel`!
+
+## Embedding your Space demo on other websites
+
+Throughout this guide, you've seen there are Gradio demos embedded. You can also do this on own website! The first step is to create a Space with the demo you want to showcase. You can embed it in your HTML code, as shown in the following self-contained example.
+
+`<iframe src="https://hf.space/gradioiframe/osanseviero/mix_match_gradio/+" frameBorder="0" height="450" title="Gradio app" class="container p-0 flex-grow space-iframe" allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"></iframe>`
```

Comment:
Formatting of this is not great

Reply:
replace with 

```bash
&lt;iframe src="https://hf.space/gradioiframe/osanseviero/mix_match_gradio/+" frameBorder="0" height="450" title="Gradio app" class="container p-0 flex-grow space-iframe" allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"&gt;&lt;/iframe&gt;
```

and include 3 backticks with bash written after them instead of the single backtick

Reply:
feel free to break it into many lines if you'd like, but this will work and will get a copy button :) 

image below as its difficult to write the instructions here without markdown rendering them! 
<img width="1086" alt="Screen Shot 2022-03-04 at 8 53 34 PM" src="https://user-images.githubusercontent.com/9021060/156805851-1b10d2ae-9908-4756-bfde-0041561235d5.png">

Reply:
Nice!!